src/biokbase/narrative/clients.py

```diff
@@ -33,20 +33,7 @@ def __init_client(client_name, token=None):
         or client_name == "job_service"
     ):
         c = execution_engine2(URLS.execution_engine2, token=token)
-    elif client_name == "job_service_mock":
-        c = JobServiceMock()
     else:
         raise ValueError('Unknown client name "%s"' % client_name)
 
     return c
-
-
-class JobServiceMock:
```

Comment:
no mention of `job_?service_?mock` anywhere else in the codebase

Reply:
I think I put that in there for either test injection or debugging usage. Still, shrug, it's not used anywhere, so...